Automattic/wp-calypso
# /themes - Add standard nav

#### Proposed Changes

* /themes: Integrate the LOHP navigation bar
* 1148-gh-Automattic/martech
* pduY5I-dl-p2

#### Testing Instructions



* Log out
* Navigate to /themes or /themes/free or /theme/archeo
* Ensure that you see the sitewide header Nav bars.
* The page should be unaffected if logged in

#### Pre-merge Checklist
